src/app/shared/services/applications-service.ts

```diff
@@ -50,13 +50,17 @@ export class ApplicationsService {
     }
 
     public deletePod(applicationName: string, podName: string): Promise<any> {
-        return requests.delete(`/applications/${applicationName}/pods/${podName}`).send().then((res) => true);
+        return requests.delete(`/applications/${applicationName}/pods/${podName}`).send().then(() => true);
     }
 
     public resourceEvents(applicationName: string, resourceUID: string, resourceName: string): Promise<models.Event[]> {
         return requests.get(`/applications/${applicationName}/events`).query({resourceName, resourceUID}).send().then((res) => (res.body as models.EventList).items || []);
     }
 
+    public terminateOperation(applicationName: string) {
+        return requests.delete(`/applications/${applicationName}/operation`).send().then(() => true);
```

Comment:
I don't see a return type on this function.  Do we need `return` here?

Reply:
Typescript will infer return type, but it is good to define it explicitly. fixed